analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -167,10 +167,16 @@ open class Npm(
             licenseNode["type"]?.textValue()
         }
 
-        return declaredLicenses.mapTo(sortedSetOf()) {
-            // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
-            // private or unpublished package under any terms", which corresponds to SPDX's "NONE".
-            if (it == "UNLICENSED") SpdxLicense.NONE else it
+        return declaredLicenses.mapTo(sortedSetOf()) { declaredLicense ->
+            when {
+                // NPM does not mean https://unlicense.org/ here, but the wish to not "grant others the right to use a
+                // private or unpublished package under any terms", which corresponds to SPDX's "NONE".
+                declaredLicense == "UNLICENSED" -> SpdxLicense.NONE
+                // NPM allows to declare non-SPDX licenses only by referencing a license file. Avoid reporting an
+                // [OrtIssue] by mapping this to a valid license identifier.
```

Comment:
Isn't the fact whether an OrtIssue is reported or not subject to evaluator rules, and thus user-specific?

Reply:
The `OrtIssue` I'm talking about has nothing to do with the `evaluator`.
It is in fact an `analyzer` issue.

Reply:
Oh, wow, I wasn't aware that we're flagging this even as an `ERROR`... thanks for reminding me of that code.

Reply:
Sure! Thanks for reviewing!